src/libs/state.js

```diff
@@ -17,6 +17,8 @@ export const authStore = Utils.atom({
   oidcConfig: {}
 })
 
+export const getUser = () => authStore.get().user
```

Comment:
Copied from libs/auth.js to allow other modules to use this without pulling in all of auth's dependencies.

Reply:
DRY
It would be better to re-export the getUser export from auth.js here so that the code (even tho it's small) lives in one place and can be changed in one place.

Reply:
Trying to avoid circular imports. auth imports from ajax. But auth could re-export this function from state.